test/tests/mesh_modifiers/sidesets_between_subdomains/between_vector_blocks.i

```diff
@@ -0,0 +1,27 @@
+[Mesh]
+  type = GeneratedMesh
+  dim = 2
+  nx = 2
+  ny = 2
+  nz = 0
+  zmin = 0
+  zmax = 0
+  elem_type = QUAD4
+[]
+
+[MeshModifiers]
+  [./subdomain_id]
+    type = AssignElementSubdomainID
+    subdomain_ids = '0 0
+                     1 2'
+  [../]
+  [./extrude]
+    type = SideSetsBetweenSubdomains
```

Comment:
Change to

[./interface]
  type = SideSetsBetweenSubdomain
  depends_on = '0'
  master_block = '0'
  paired_block = '1 2'
  new_boundary = 'in_between'
[../]